NEWS.md

```diff
@@ -0,0 +1,68 @@
+
+This document summarizes the most important changes in recent Spicy releases.
+release. For an exhaustive list of all changes, see the ``CHANGES`` file.
+
+Spicy 1.1
+=========
+
+New Functionality
+-----------------
+
+  * GH-844: Add support for `&size` attribute to unit `switch` statement.
+
+  * GH-26: Add `%skip`, `%skip-pre` and `%skip-post` properties
+    for skipping input matching a regular expression before any
+    further input processing takes place.
+
+  * Extend library functionality provided by the `spicy` module:
+
+    - `crc32_init()/crc32_add()` compute CRC32 checksums.
+    - `mktime()` creates a `time` value from individual components.
+    - `zlib_init()` initializes a `ZlibStream` with a given window bits argument.
+    - `Zlib` now accepts a window bits parameter.
+
+  * Add a new `find()` method to units for that searches for a `bytes`
+    sequence inside their input data, forward or backward from a given
+    starting position.
+
+  * Add support for `&chunked` when parsing bytes data with `&until`
+    or `&until_including`.
+
+  * Add `encode()` method to `string` for conversion to `bytes`.
+
+  * Extend parsing of `void` fields:
+
+      - Add support for `&eod` to skip all data until the end of the
+        current input is encountered.
+
+      - Add support for `&until` to skip all data until a deliminator
+        is encountered. The deliminator will be extracted from the
+        stream before continuing.
+
+  * Port Spicy to Apple silicon.
+
+  * Add Dockerfile for OpenSUSE 15.2.
+
+Changed Functionality
+---------------------
+
+  * Reject `void` fields with names.
+  * Lower minimum required Python version to 3.2.
```

Comment:
I think this isn't new since 1.0.0, but the commit in question instead fixed a patch introduced since then which inadvertently raised the minimum required version.

Reply:
Yeah, but compared to the 1.0, we still lowered it, it used to be 3.4.

Reply:
Ah right.